Code/GraphMol/FindStereo.cpp

```diff
@@ -1014,9 +1014,10 @@ std::vector<StereoInfo> runCleanup(ROMol &mol, bool flagPossible,
     const bool includeChirality = false;
     const bool includeIsotopes = false;
     const bool breakTies = false;
```

Comment:
Looks good, perhaps a bit of documentation of what the rankFragmentAtoms is doing here to find the priorities would be useful.